trade-finance-manager-ui/templates/case/deal/deal.njk

```diff
@@ -1,15 +1,27 @@
 {% from "govuk/components/details/macro.njk" import govukDetails %}
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "moj/components/banner/macro.njk" import mojBanner %}
 {% import './_macros/eligibility-criteria-answers.njk' as eligibilityCriteriaAnswers %}
 {% import './_macros/facilities-table.njk' as facilitiesTable %}
 
 {% extends "case/case.njk" %}
 
 {% block sub_content %}
+  {% if hasAmendmentInProgress === true %}
```

Comment:
Does it need to be "true"?

Reply:
unfortunately yes. for some reason nunjucks doesn't always detect boolean like this `{% if hasAmendmentInProgress %}`

So I had to explicitly add it